code-challenges.js

```diff
@@ -3,8 +3,21 @@
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-
-
+// function that takes in a certain number of indexes
+let fibonacci = (number) =>
+{
+// if the number inputed is 1 return [1,1]
+  if (number === 1) {
+    return [1,1]
+  } else {
+// variable  that includes the fibonacci formula and pushes the answer into solution variable
+    let solution = fibonacci(number - 1)
+    solution.push(solution[solution.length - 1] + solution[solution.length - 2])
+    return solution
+  }
+}
+
+ console.log(fibonacci(10))
```

Comment:
ooooWEE, Nice work! This is a minor issue, but I think you're output is one integer too much!  No worries here, I love your refactor.